index.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html lang="en-us">
+
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/edb362f252.js" crossorigin="anonymous"></script>
+    <title>HTML Forms</title>
+</head>
+
+<body>
+    <header>
+        <div class="sign-in-button">
+            <button type="submit" name="submit">
+                <i class="fas fa-lock"></i> Sign in</button>
+        </div>
+        <div>
+            <p>Already have an Intuit account?</p>
+            <a href="#">I forgot my User ID or Password</a>
+        </div>
+    </header>
+
+    <main>
+        <div>
+            <div class="intuit">
+                <img src="assets/intuit_logo.png" alt="intuit logo" width="195" height="65" />
+            </div>
+
+            <div class="logos">
+                <img src="assets/turbotax_logo.png" alt="turbotax logo" width="135" height="30" />
+                <img src="assets/quickbooks_logo.png" alt="quickbooks logo" width="160" height="30" />
+                <img src="assets/mint_logo.png" alt="mint logo" width="85" height="30" />
+            </div>
+        </div>
+        <div class="container">
+            <div>
+                <h1>Create an Intuit account</h1>
+                <p>One account for everything Intuit, including Mint.</p>
+                <a class="learn-more" href="url">Learn more</a>
+                <form>
+                    <label for="email">Email address</label>
+                    <input type="email" name="email" id="email" />
+                    <label for="phone">Phone (recommended)</label>
+                    <input type="number" name="phone" id="phone" />
+                    <p class="form-info">Standard call, messaging or data rates may apply.</p>
+                    <label for="password">Password</label>
+                    <input type="text" name="password" id="password"/>
+                    <button type="submit" name="submit">
+                        <i class="fas fa-lock"></i> Create Account</button>
+                </form>
+                <p class="disclaimer"><i>By clicking Create account, you agree to our <a href="#">Terms</a>
+                        and have read and acknowledge our <a href="#">Privacy Statement.</a></i></p>
+                <div class=graybox>
+                    <p>Invisible reCPATCHA by Google <a href="#">Privacy Policy</a> and
```

Comment:
The value of attribute [ class ] must be in double quotes. So line number 53 here class graybox should be in double quotes.